azure_jumpstart_ag/artifacts/settings/ESA/configPod.yaml

```diff
@@ -0,0 +1,35 @@
+kind: Deployment
```

Comment:
What's the purpose of this configPod? This would be part of the end deployment, like the EdgeAI pod that users this PV through a PVC

Reply:
Test app to check if ESA is working